wiki/Play_style/Tapx/id.md

```diff
@@ -0,0 +1,29 @@
+---
+stub: true
+tags:
+  - tablet tap
+  - tap
+  - tekanan
+  - tablet
+  - streaming with tablet
+  - play style
+  - gaya bermain
+---
+
+# Tapx
+
+*Catatan: Gaya bermain ini membutuhkan sebuah graphic tablet.*
+
+**Tapx** adalah sebuah gaya tapping bagi pemain yang [hanya menggunakan tablet](/wiki/Play_style#tablet-only) untuk bermain pada mode [osu!](/wiki/Game_mode/osu!) di mana pemain menekan circle, meluncurkan slider, dan memutar spinner dengan menekankan pena tablet ke tablet, bersama dengan tombol keyboard/tablet kedua untuk membantu pemain dalam melakukan [stream](/wiki/Beatmap/Pattern/Stream) dan [note-note bertumpuk](/wiki/Mapping_Techniques/Stack). Metode untuk bermain ini cukup sulit untuk kebanyakan pemain dan tidak terlalu popular bagi kebanyakan pemain tablet.
```

Comment:
```suggestion
**Tapx** adalah sebuah gaya tapping bagi pemain [osu!](/wiki/Game_mode/osu!) yang [hanya menggunakan tablet](/wiki/Play_style#tablet-only) untuk bermain. Di mana pemain menekan circle, meluncurkan slider, dan memutar spinner dengan menekankan pena tablet ke tablet, bersama dengan tombol keyboard/tablet kedua untuk membantu pemain dalam melakukan [stream](/wiki/Beatmap/Pattern/Stream) dan [stack](/wiki/Mapping_Techniques/Stack). Metode untuk bermain ini cukup sulit bagi kebanyakan pemain dan tidak terlalu popular bagi kebanyakan pemain tablet.
```

Reply:
suggesting to reduce repeat words and it's should be called stack consider mania players also calling stack.